src/components/table/_macro.njk

```diff
@@ -37,18 +37,7 @@
                             {% if td.form is defined and td.form %}
                                 <form action="{{ td.form.action }}" method="{{ td.form.method | default('POST')}}">
                                     {{
```

Comment:
This would need to be removed I think

Reply:
Cool, I'll get on it tomorrow morning

Reply:
Looking at it, not sure what you mean - do you want to remove the form action altogether?

Reply:
Hi, you have {{ {{onsButton(td.form.button)}} }} so you need to remove the outer {{ }}

Reply:
Ah, gotcha, done